tests/acceptance/features/coreApiShareOperationsToShares2/shareAccessByID.feature

```diff
@@ -1,5 +1,8 @@
 @api
 Feature: share access by ID
+  As a user
+  I want to access a share by its id
+  So that I can confirm it works
```

Comment:
This is OK for now.
We can think about what to say for back-end features that are really API end-points that are to make it easier for appps.
Maybe:
```suggestion
  As an API consumer (app)
  I want to access a share by its id
  So that the app can more easily manage shares.
```